Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6-onbuild
```

Comment:
Well you'd still have golang in the container in the end and the other way
you can make it just install go to build the binary and make the container
only have the build artifacts after, no golang etc

On Sunday, March 13, 2016, Aidan Feldman <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/cloudflare/redoctober/pull/131#discussion_r55943728>:
>
> > @@ -0,0 +1,10 @@
> > +FROM golang:1.6-onbuild
>
> Any reason to do that instead of a gloang:*-alpine image?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudflare/redoctober/pull/131/files#r55943728>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>